pyvista/plotting/plotting.py

```diff
@@ -1387,6 +1387,17 @@ def add_mesh(self, mesh, color=None, style=None, scalars=None,
             If set to ``True``, then the number of unique values in the scalar
             array will be used as the ``n_colors`` argument.
 
+        silhouette : dict, bool, optional
+            If set to ``True``, plot a silhouette highlight for the mesh. This
+            feature is only available for a triangulated ``PolyData``.
+            As a ``dict``, it contains the properties of the silhouette to display:
+
+                * ``color``: ``str`` or 3-items ``list``, color of the silhouette
+                * ``line_width``: ``float``, edge width
+                * ``opacity``: ``float`` between 0 and 1, edge transparency
+                * ``feature_angle``: If ``True``, display sharp edges
+                * ``decimate``: ``float`` between 0 and 1, level of decimation
```

Comment:
Did you change the `None` case to be undocumented on purpose? Probably not critical as the example uses it.

Reply:
To make it consistent with `silhouette`'s behaviour and `None` is not tested explicitly anymore.

Reply:
Ah, I see, you're only handling floats.

But then one thing to watch out is that someone passing `decimate=0` or `decimate=1` would not enter the corresponding `if` branch. This might not be a problem, because `decimate=0` would mean "no decimation" anyway. And I presume `decimate=1` would be nonsense. Or can that be intended input? I've never had anything to do with decimation. If there's a risk of passing int literals for `decimate` then it might be safer to add `int` to the `isinstance` check.